doc/nrf/known_issues.rst

```diff
@@ -161,6 +161,14 @@ NCSDK-13895: Build failure for target Thingy:91 with secure_bootloader overlay
 Other issues
 ============
 
+.. rst-class:: v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-14312: During execution, in rare cases, the :c:func:`nrf_recv()` function would crash.
+
+.. rst-class:: v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+
+NCSDK-13360: When a socket that is in a :c:func:`nrf_recv()` loop is closed by another thread, it will eventually access the packet handler in the socket context causing a crash since the socket context was destroyed by the :c:func:`nrf_close()` call.
```

Comment:
`The socket context crashes when closed by another thread` -> `Closing a socket while another thread is receiving data from it may crash`

"When a socket that is in a nrf_recv() function ~~loop~~", it does not need to be a loop, it just needs to be in a recv() call